src/NServiceBus.Core.Tests/Pipeline/Outgoing/OutgoingPublishContextTests.cs

```diff
@@ -56,7 +39,7 @@ public void ShouldNotMergeOptionsToParentContext()
 
             var parentContext = new RootContext(null,null,null);
 
-            new OutgoingPublishContext(message, options, parentContext);
+            new OutgoingPublishContext(message, "id", options.OutgoingHeaders, options.Context, parentContext);
```

Comment:
Nitpick: `message-id` or similar would be a better value to avoid the reader guessing